frontend/src/components/PodcastingInfo.tsx

```diff
@@ -0,0 +1,63 @@
+import { Boostagram } from "src/types";
+
+function PodcastingInfo({ boost }: { boost: Boostagram }) {
+  return (
+    <>
+      {boost?.message && (
```

Comment:
Everywhere `boost` has a `?` afterwards - this should not be needed

Reply:
Oops yes ✅